.github/workflows/build.yml

```diff
@@ -53,3 +53,30 @@ jobs:
           }}/actions/runs/${{ github.run_id }}/cancel
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  test:
+    runs-on: ${{ matrix.os }}
+    permissions:
+      contents: read
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.x
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+      - name: Install pipenv
+        run: pip install pipenv
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: "1.16"
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+      - name: Run integration tests
+        run: yarn run integ
```

Comment:
Currently the integ tests are running as part of the build task:

https://github.com/cdk8s-team/cdk8s/blob/7ae13bffc149f4770ff6848665578dcbd2103c88/.projen/tasks.json#L52-L58

How come this doesn't need a `lerna run build` before?

Reply:
We don't need to run a build for the integ tests because they don't rely on any source code within this repo (they're all bash scripts).